### DISCUSSION TITLE
POS rewards in Firefly current wallet.

### DISCUSSION THREAD

**Discussion by Wdmoore01** [iotaledger/firefly#8574] [Feedback]
I found the Firefly wallet that claimed online to give Point of Staking Rewards.  I have been waiting for 1.5 months and so far nothing shows in the total tokens, nor in the Rewards section of the wallet. What do I need to do to fix this Please?  I went to Discord and it fails to allow me access to the IOTA support team there as well. Please respond.

**Comment by begonaalvarezd** [comment]
duplicate of https://github.com/iotaledger/firefly/issues/8549
